### PR TITLE
Hack to prevent issue #1208 in Gahreesen age

### PR DESCRIPTION
**Pull request by Hazado**
Issue  https://github.com/H-uru/Plasma/issues/1208
Prevents the OnNotify from running before OnServerInitComplete is done.

Arriving in Gahreesen from the Bahro Pole in Relto seems to run through the entryTrigger region and triggers the subworld entry.
This causes the avatar to fall out of the world because they are not anywhere with solid ground.

### CODE REVIEW

File: Scripts/Python/grsnNexusBookMachine.py [hunk 1]
```diff
@@ -81,6 +81,7 @@
 kElevMoving = 1
 kElevUp = 2
 elevatorStatus = kElevDown
+serverInitDone = False
```
Comment (Hoikas) [inline Scripts/Python/grsnNexusBookMachine.py:84]: A class member would probably be better than another global.

**Review by Hoikas — COMMENTED** [review]
This is a bit of a hack that leaves the underlying problem unsolved. I would be curious to see the result of [this experiment](https://discord.com/channels/282045216221822978/282046743338811392/1141488858056372254) before accepting this.

**Comment by Hazado** [comment]
> This is a bit of a hack that leaves the underlying problem unsolved. I would be curious to see the result of [this experiment](https://discord.com/channels/282045216221822978/282046743338811392/1141488858056372254) before accepting this.

I changed the spawnpoint names for the nexus elevators and the issue still happened.
I dont think the linked experiment will change whats happening here.
The avatar does arrive at the right spawn point, but ends up in a subworld.

**Comment by Hoikas** [comment]
A real fix hasn't exactly been forthcoming, so we might as well merge this hack for now.

**Pull request merged**